content/workpackages/05_Capacity_and_Recognition/index.md

```diff
@@ -6,7 +6,10 @@ shorthand: WP5
 weight: 600
 ---
 
-This WP will align, adapt, extend, and curate research software quality training resources across research communities, building around the RSQkit, and to deliver a recognition framework for rewarding scientists and trainers for their contributions to research software, adapting existing platforms aligned to larger research attribution initiatives.
+Software and computing are becoming more and more essential tools for research, which are used on a daily basis. Yet proper acknowledgement of the work that is going into the development of such programmes is not yet adequate. Work package 5 has launched an analysis of existing award schemes and trainings. From this a a set of guidelines will be established. Not only is it imperative to accordingly educate (young) people entering the world of software, but also to then recognise the effort they have put into creating powerful modern tools for data analysis.
+
+While this is a common framework, training methods will be tailored to the respective research communities (astronomy & particle physics, photon & neutron science, environmental science, life sciences, social sciences & humanities), so that developers can focus their learning on the most important tools for their discipline.
```

Comment:
Suggested rewording: "While the work package will aim at developing a common framework, training methods..."